lf/demo/demo_robot.lf

```diff
@@ -57,6 +92,8 @@ reactor Controller {
     state backing_distance:int(0);
     state turning_distance:int(0);
     
+    // Here you are describing the entire behavior you will have to put into the App container callback. Luos will call this callback in case of "pressed" or "bumped" information input and let you know wich one you are dealing with.
+    // As I understand it the -> represent a reaction message to another container or the target in Luos vocabulary
```

Comment:
In the coordination language (not the target language), `->` can either be a connection statement if there is a port on both sides or, if used in a reaction declaration, it indicates the potential effects of the reaction (potentially write to certain output ports, schedule some actions, etc.).